config/config_test.go

```diff
@@ -105,6 +106,10 @@ func TestGetLintingRules(t *testing.T) {
 			confPath:       "testdata/enable2.toml",
 			wantRulesCount: 2,
 		},
+		"var-naming configure error": {
+			confPath: "testdata/varNamingConfigureError.toml",
+			wantErr:  `cannot configure rule: "var-naming": invalid argument to the var-naming rule. Expecting a allowlist of type slice with initialisms, got string`,
```

Comment:
nit:

```suggestion
			wantErr:  `cannot configure rule: "var-naming": invalid argument to the var-naming rule. Expecting an allowlist of type slice with initialisms, got string`,
```

Reply:
Thanks for this. Changing the error message for the `var-naming` rule is not the topic of this PR. It should be changed here in a separate PR: https://github.com/mgechev/revive/blob/3d1115dacd11e96bb934ac915cb2f398d9ae51df/rule/var_naming.go#L276.